packages/components/organisms/f-offers/src/components/_tests/Header.test.js

```diff
@@ -0,0 +1,122 @@
+import Vuex from 'vuex';
+import { createLocalVue, mount } from '@vue/test-utils';
+import MediaElement from '@justeat/f-media-element';
+import { VueI18n } from '@justeat/f-globalisation/src';
+import { VUEX_MODULE_NAMESPACE_OFFERS } from '../../store/types';
+import Header from '../Header.vue';
+import tenantConfigs from '../../tenants';
+
+const localVue = createLocalVue();
+
+localVue.use(VueI18n);
+localVue.use(Vuex);
+
+const mockLocale = 'en-GB';
+
+const i18n = {
+    locale: 'en-GB',
+    messages: {
+        'en-GB': tenantConfigs['en-GB'].messages
+    }
+};
+
+const i18nMocker = key => key.split('.').reduce((acc, current) => acc[current], tenantConfigs[mockLocale].messages);
+
+describe('Header.vue', () => {
+    describe('ssAuthenticated: false', () => {
+        let wrapper;
+
+        beforeEach(() => {
+            // Arrange
+            const getters = {
+                isAuthenticated: () => false,
+                friendlyName: () => 'Matthew'
```

Comment:
Is `friendlyName` always defined with a value? Could it be `undefined` or `null` for example?

Reply:
good point I will make sure I am accounting for this, thanks